### PR TITLE
specs: guaranteed gas market cleanup

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Adds in an additional section explaining that the gas burn helps with sybil resistance, to prevent network spam on L2. It was not clear that this was the case when reading the doc. Also clean up the pseudocode.

### CODE REVIEW

File: specs/guaranteed-gas-market.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ at a multiple of the target, we enable deposits to temporarily use more L2 gas a
 # the ability of the account to afford this gas.
 BASE_FEE_MAX_CHANGE_DENOMINATOR = 8
 ELASTICITY_MULTIPLIER = 4
-MAX_RESOURCE_LIMIT = 8,000,000
+MAX_RESOURCE_LIMIT = 8_000_000
```
Comment (maurelian) [inline specs/guaranteed-gas-market.md:59]: There is another occurrence of the `,` delimiter in this document on line 45, and quite a few in other locations, so I think we should stick with that for now or change them all.

image

**Review by maurelian — APPROVED** [review]
LGTM once my comment is resolved.

File: specs/guaranteed-gas-market.md (hunk 1 above)
Comment (tynes) [reply specs/guaranteed-gas-market.md:59]: Good catch, I think having a comma in pseudocode looks weird, I'd prefer to change it everywhere but open to leaving it if people don't want the diff

Comment (trianglesphere) [reply specs/guaranteed-gas-market.md:59]: We can check those numbers, but I personally prefer commas in prose.

Comment (maurelian) [reply specs/guaranteed-gas-market.md:59]: I see now, @tynes is right, ie. this is inside a code block which is not obvious from the diff. 
The other instances in the screenshot are prose.

Resolving this thread.

**Pull request merged**